netci.groovy

```diff
@@ -137,6 +137,11 @@ branchList.each { branchName ->
             // Add the unit test results
             Utilities.addXUnitDotNETResults(newJob, 'bin/tests/**/testResults.xml')
 
+            // Unix runs take more than 2 hours to run, so we set the timeout to be longer.
```

Comment:
What do the Windows runs take?  If it's much less, do we understand why and have issues open to address?